governance/polls/meta/poll-tags.json

```
@@ -5062,5 +5062,17 @@
   "1254": [
     "high-impact",
     "spark"
+  ],
+  "1504": [
+    "medium-impact",
```

we would probably go with high-impact for this one

```suggestion
    "high-impact",
```